### PR TITLE
Add analytics

### LINKED ISSUE #39: Add Google Analytics tracking
Could we please add GA script and connect it to our main properly please?

@iamsolankiamit @sradevski @chrisys

### PR DESCRIPTION
**Pull request by sradevski**
Fixes: #39
Change-type: minor
Signed-off-by: Stevche Radevski <stevche@balena.io>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,9 +40,11 @@
     "@types/node": "^13.11.1",
     "@types/react": "^16.9.33",
     "@types/react-dom": "^16.9.6",
+    "analytics-client": "^0.5.3",
     "arch": "^2.1.1",
     "balena-sdk": "^13.0.0",
     "lodash-es": "^4.17.15",
+    "mixpanel-browser": "^2.35.0",
```
Comment (sradevski) [inline package.json:48]: This is in devdependencies in analytics-client but it fails to start if not added, so we need to move it to dependencies.

**Review by iamsolankiamit — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (roman-mazur) [reply package.json:48]: Will fix, thanks.

**Review by roman-mazur — APPROVED** [review]

**Comment by sradevski** [comment]
@balena-ci rebase

**Comment by sradevski** [comment]
@balena-ci rebase

**Pull request merged**